code/datums/diseases/chronic_illness.dm

```diff
@@ -86,3 +93,13 @@
 					affected_mob.investigate_log("has been dusted / deleted by [name].", INVESTIGATE_DEATHS)
 					affected_mob.ghostize(can_reenter_corpse = FALSE)
 					qdel(affected_mob)
+
+//Bubbers addition - Costly warning if it sneaks through to stage four.
+/datum/disease/chronic_illness/update_stage(new_stage)
+	. = ..()
+	if(new_stage == 4 && being_stealthy)
+		to_chat(affected_mob, span_danger("[pick("You feel as though your atoms are accelerating in place.", "You feel like you're being torn apart!")]"))
+		affected_mob.emote("scream")
+		affected_mob.adjustBruteLoss(10, updating_health = FALSE)
+		affected_mob.updatehealth()
```

Comment:
adjustBruteLoss() already comes with update_health, any reason we'd need to split it up like this?
```suggestion
		affected_mob.adjustBruteLoss(10)
```

Reply:
Yeah, that was a moment of me reading the code and it not fully clicking what was going on. Fixed.